datamodel/create_roles.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+
+import re
+from argparse import ArgumentParser, BooleanOptionalAction
+from typing import Optional
+
+from pgserviceparser import service_config
+
+try:
+    import psycopg
+except ImportError:
+    import psycopg2 as psycopg
+
+
+def get_db_identifier(pg_service: str, modelname: str):
```

Comment:
```suggestion
def get_db_identifier(pg_service: str, modulename: str):
```
don't we talk about a module rather than a model?

Reply:
good point, will alter it